src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -249,6 +250,37 @@ class CreateTableStatementSegment(BaseSegment):
     )
 
 
+@redshift_dialect.segment(replace=True)
+class InsertStatementSegment(BaseSegment):
+    """A `INSERT` statement.
+
+    Redshift has two versions of insert statements:
+        - https://docs.aws.amazon.com/redshift/latest/dg/r_INSERT_30.html
+        - https://docs.aws.amazon.com/redshift/latest/dg/r_INSERT_external_table.html
+    """
+
+    # TODO: This logic can be streamlined. However, there are some odd parsing issues.
+    # See https://github.com/sqlfluff/sqlfluff/pull/1896
+
+    type = "insert_statement"
+    match_grammar = Sequence(
+        "INSERT",
+        "INTO",
+        Ref("TableReferenceSegment"),
+        OneOf(
+            OptionallyBracketed(Ref("SelectableGrammar")),
+            Sequence("DEFAULT", "VALUES"),
```

Comment:
I don't see a test that uses `DEFAULT VALUES`. Should there be?

Reply:
I believe this already exists [here](https://github.com/sqlfluff/sqlfluff/pull/1896/files#diff-4a8ae40a1739111fbb9a0b5f9be034d4d56d1aa05ec975398112a3049c9784e9R25-R26).